spec/support/factory_girl.rb

```diff
@@ -6,6 +6,8 @@
     if ENV['OSEM_FACTORY_LINT'] != 'false'
       mock_commercial_request
       FactoryGirl.lint
```

Comment:
Now that we are doing it in a rake task I think that we can remove this file together with the `OSEM_FACTORY_LINT` as there is no point in linting the factories before each suite.